install

```diff
@@ -390,6 +390,11 @@ Dir.chdir HOMEBREW_REPOSITORY do
   end
 end
 
+# Tap linuxbrew/xorg to ensure :x11 dependencies are available
+# Ensure $HOMEBREW_PREFIX/bin is in PATH so Git is usable after installation
+system "env", "PATH=#{HOMEBREW_PREFIX}/bin:#{ENV["PATH"]}", "brew", "tap",
+  "linuxbrew/xorg" unless mac?
```

Comment:
`system` takes a hash table of environment variables as its first argument.
https://ruby-doc.org/core-2.2.0/Kernel.html#method-i-spawn
Slightly modified the comment to improve clarity (to me, anyway).
```ruby
# Tap linuxbrew/xorg to ensure :x11 dependencies are available.
# Ensure $HOMEBREW_PREFIX/bin is in PATH so the brewed git is usable.
system({"PATH" => "#{HOMEBREW_PREFIX}/bin:#{ENV["PATH"]}"},
    "brew", "tap", "linuxbrew/xorg") unless mac?
```